Source/Stationeers.Addons.Patcher/Core/Patchers/MonoPatcher.cs

```diff
@@ -121,14 +125,13 @@ public bool IsPatched()
         private void GetInstanceAssemblies(string installInstance)
         {
             // This is kind of verbose, might need to be rewritten in a more concise manner
-            string installResourcesDir = null;
-            if(installInstance == Constants.GameExe)
+            if (installInstance == Constants.GameExe)
             {
-                installResourcesDir = Constants.GameResourcesDir;
+                _installResourcesDir = Constants.GameResourcesDir;
             }
-            else if(installInstance == Constants.ServerExe)
+            else if (installInstance == Constants.ServerExe)
             {
-                installResourcesDir = Constants.ServerResourcesDir;
+                _installResourcesDir = Constants.ServerResourcesDir;
             }
             System.Diagnostics.Debug.Assert(!string.IsNullOrEmpty(installResourcesDir), "Invalid install dir!");
```

Comment:
Compiler errors on this and the following two lines since `installResourcesDir` was renamed incompletely, I believe; I successfully compiled after changing `installResourcesDir` to `_installResourcesDir` as the above changes did.

Reply:
that makes sense, my bad, i'll upload a patch this afternoon